Fixed hair

chikincheese opened the pull request:
...by forcing it to be in a new function. No, I don't know why.

Ugre101 commented:
When looking at your fix, the problem jumped to my head.
player.Haircolor = document.getElementById("haircolor").value; 
Under Begin needs to be
player.Face.Haircolor = document.getElementById("haircolor").value;

chikincheese commented:
So, ensured that lastname is consistent over all files (it wasn't), added in your fixed hair mechanics, and added in a concept for alternate race shifting.
